vagga.yaml

```diff
@@ -0,0 +1,26 @@
+containers:
+  pip-freeze:
+    setup:
+      - !Alpine v3.4
+      - !Py3Install
+        - pip
+        - 'Django >= 1.9,<=1.10.5'
+        - 'gunicorn >=19.5,<=16.6.0'
+        - 'whitenoise >= 3.2,<=3.2.3'
+        - 'django-environ >= 0.4,<=0.4.1'
```

Comment:
ok
so it'd be better to write for example 
gunicorn >=19.5,<=19.6.0'
'whitenoise >= 3.2,<=3.3'
'django-environ >= 0.4,<=0.5'

right?